### ISSUE TITLE
How to show some illustration after complete train model ?

### ISSUE THREAD

**Issue by Houangnt** [crux82/ganbert-pytorch#8]

**Comment by crux82** [comment]
Sorry, what do you mean with Demo?

**Comment by Houangnt** [comment]
> Sorry, what do you mean with Demo?

I want to show the results after training model on unlabeled dataset with actual data (Example : What does CNN stand for ? -> Result after classified is: ABBR:exp) . Also, Whether i can put some new data has not yet encountered for model predict ?

**Comment by crux82** [comment]
In the python book you can find

    # Evaluate data for one epoch
    for batch in test_dataloader:
        
        # Unpack this training batch from our dataloader. 
        b_input_ids = batch[0].to(device)
        b_input_mask = batch[1].to(device)
        b_labels = batch[2].to(device)
        
        # Tell pytorch not to bother with constructing the compute graph during
        # the forward pass, since this is only needed for backprop (training).
        with torch.no_grad():        
            model_outputs = transformer(b_input_ids, attention_mask=b_input_mask)
            hidden_states = model_outputs[-1]
            _, logits, probs = discriminator(hidden_states)
            ###log_probs = F.log_softmax(probs[:,1:], dim=-1)
            filtered_logits = logits[:,0:-1]

If you just take the maximum value in `filtered_logits` you have the class.

Hope it helps

Danilo
